spec/defines/vhost_spec.rb

```diff
@@ -2171,12 +2171,12 @@
                 'enabled bob',
               ],
             )
+          end
 
-            it {
-              expect(subject).to contain_concat__fragment('rspec.example.com-apache-userdir')
-                .with(content: %r{^\s+UserDir disabled$})
-                .with(content: %r{^\s+UUserDir enabled bob$})
```

Comment:
Looks like there was already a typo in `UUserDir` here, which is proof the test didn't ran.